tests/test_4taskrun.py

```diff
@@ -6,6 +6,7 @@
 import signal
 import time
 from .context import WDL
+from testfixtures import log_capture
```

Comment:
look like we need testfixtures in requirements.dev.txt

Reply:
👍